tools/analyzer_plugin/lib/src/util/fix.dart

```diff
@@ -7,7 +7,7 @@ import 'package:analyzer_plugin/utilities/change_builder/change_builder_dart.dar
 
 Future<SourceChange> buildFileEdit(
   ResolvedUnitResult result,
-  FutureOr<void> Function(DartFileEditBuilder builder) buildFileEdit,
+  void Function(DartFileEditBuilder builder) buildFileEdit,
```

Comment:
This typing seemed to be a typo; calls to `buildFileEdit` aren't awaited, and all the existing `buildFileEdit`s were synchronous. This signature now matches that of `buildGenericFileEdit`